test/e2e/data/e2e_conf.yaml

```diff
@@ -26,19 +26,30 @@ images:
     loadBehavior: tryLoad
   - name: quay.io/jetstack/cert-manager-controller:v1.12.2
     loadBehavior: tryLoad
-  - name: registry.k8s.io/cluster-api/cluster-api-controller:v1.4.4
+  - name: registry.k8s.io/cluster-api/cluster-api-controller:v1.5.0
     loadBehavior: tryLoad
-  - name: registry.k8s.io/cluster-api/kubeadm-bootstrap-controller:v1.4.4
+  - name: registry.k8s.io/cluster-api/kubeadm-bootstrap-controller:v1.5.0
     loadBehavior: tryLoad
-  - name: registry.k8s.io/cluster-api/kubeadm-control-plane-controller:v1.4.4
+  - name: registry.k8s.io/cluster-api/kubeadm-control-plane-controller:v1.5.0
     loadBehavior: tryLoad
 
 providers:
   - name: cluster-api
     type: CoreProvider
     versions:
-      - name: v1.4.4 # latest published release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
-        value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.4.4/core-components.yaml"
+      - name: v1.2.0 # earliest published release in the v1beta1 series; this is used for v1beta1 old --> v1beta1 latest clusterctl upgrades test only.
```

Comment:
@Ankitasw why was this changed to v1.2.0? I think this might be the source of the issue for the upgrade tests.

Reply:
We were using v1.4.4, I have updated this version to v1.5.0, if you see the below lines in same file, added v1.2.0 only for clusterctl upgrade tests